src/Java.Interop/Java.Interop/JniEnvironment.Types.cs

```diff
@@ -47,6 +48,8 @@ public static unsafe JniObjectReference FindClass (string classname)
 					return r;
 				}
 
+				// NativeMethods.java_interop_jnienv_exception_describe (info.EnvironmentPointer);
```

Comment:
dead code?

Reply:
Yes, but not entirely; from the commit message:

> The attempt to retain useful Java-side exceptions in 89a5a22
> proved to be incomplete. Add a comment to invoke
> JNIEnv::ExceptionDescribe(). We don't always want this
> to be present, but when we do want it…